build.gradle

```diff
@@ -63,13 +64,16 @@ ext {
                 fragmentktx: "androidx.navigation:navigation-fragment-ktx:$navigationVersion",
                 ui: "androidx.navigation:navigation-ui:$navigationVersion",
                 uiktx: "androidx.navigation:navigation-ui-ktx:$navigationVersion",
-                dynamicfeatures: "androidx.navigation:navigation-dynamic-features-fragment:$navigationVersion"
+                dynamicfeatures: "androidx.navigation:navigation-dynamic-features-fragment:$navigationVersion",
+                navtesting: "androidx.navigation:navigation-testing:$navigationVersion",
+                fragmenttesting: "androidx.fragment:fragment-testing:$fragmentVersion"
```

Comment:
I guess these should go under `test`, not `androidx`? I _do_ currently have Android X dependencies down there, as well. I'll admit that the layout of this `ext` block makes no sense. I just _did some stuff_. We can re-organize things if you can think of a better standardization. Currently I am trying to minimize the average length of `dep.foo.bar.whatever`, while still having descriptive and meaningful phrasing. A linguistic optimization challenge of sorts, information science at its core! Aye.